index.html

```diff
@@ -0,0 +1,116 @@
+<head>
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Thinus Capstone</title>
+    </head>
+    <body class="BGColor">
+        <!-- col-1 will be used instead of col-xs-1 since bootstrap 5 no longer supports xs screen size -->
+        <nav class="navbar-fluid">
+            <div class="MobileMenu hidden">
+                <ul class="MobileNavBottom row col-12 col-sm-12">
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="pages/about.html">About</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="index.html">Program</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Join</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Sponsor</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">News</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Warframe Lore</a></li>
+                </ul>
+            </div>
+            <div class="NavTop container-fluid">
+                <ul class="row col-lg-4 col-4 col-sm-4 offset-sm-8">
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/fbicon.png" alt="Facebook Icon"></a></li>
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/twittericon.png" alt="Twitter Icon"></a></li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">English</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">My Page</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">Logout</li>
+                </ul>
+            </div>
+            <div class="container-fluid">
+                <ul class="NavBottom row col-lg-11 col-11 col-sm-11 offset-1">
+                    <li class="col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="col-lg-5 offset-lg-11 col-5 col-sm-5" src="images/lotusIcon.jpg" alt="lotusIcon"></a></li>
+                    <li class="Bnav AboutColor col-lg-1 col-1 col-sm-1 offset-3"><a href="pages/about.html">About</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="index.html">Program</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">Join</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">Sponsor</a></li>
+                    <li class="Bnav Color col-lg-1 col-1 col-sm-1"><a href="#">News</a></li>
+                    <li class="Bnav Color Lore col-lg-2 col-2 col-sm-2"><a href="#">Warframe Lore</a></li>
+                </ul>
+            </div>
```

Comment:
- [ ] `required`
- [ ] As the previous reviewer requested
 * [x]  Kindly ensure that you stick as much as possible to the original design. The background color that you have used for this top bar does not match the one that is provided in the Behance design.
 * [x]  Also, kindly ensure that the nav menu links also have a similar design to that that is in the Behance design.
> 
 **This is how your design looks like:**
> 
 ![image](https://user-images.githubusercontent.com/86318284/205091193-a3c2b6bd-4bff-4f6f-af2a-556c15c04e0a.png)
> 
 **This is the desired design**
> 
 ![image](https://user-images.githubusercontent.com/86318284/205091292-88ecb04f-a6e9-4d17-b0e9-ecd05b1a7e5c.png)

Reply:
These two images are both from the previous reviewer and I did fix them